scripts/windows/test/Setup-Env.ps1

```diff
@@ -76,7 +82,15 @@ If ($osEdition -eq "IoTUAP")    # Windows IoT Core - update iotedge
         }
         
         Write-Host "Cleanup existing containers..."
-        docker -H npipe:////./pipe/iotedge_moby_engine rm -f $(docker -H npipe:////./pipe/iotedge_moby_engine ps -aq) 2>null
+        try {
+            $residualModules = $(docker -H npipe:////./pipe/iotedge_moby_engine ps -aq)
+            if($residualModules.Length -gt 0) {
+                docker -H npipe:////./pipe/iotedge_moby_engine rm -f $residualModules
+            }
+        }
+        catch {
+            Write-Host "Cleanup existing containers failed. Nothing to clean."
```

Comment:
Remove "Nothing to clean"; and should at least print the exception caught.

Reply:
Done.